.github/ISSUE_TEMPLATE/propose-a-new-standard-name.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Propose a new Standard Name
```

Comment:
I think there is potential for some confusion with this name as the `about:` section indicates this template could be used for amendments as well as suggesting a new name. Also as stated below "it is fine to group together a number of proposals in a single GitHub issue" so ideally the title could reflect that multiple names might be being proposed. As-is, something like:

```suggestion
name: Propose new Standard Name(s) or amendment(s) to the current table of names
```

would be more appropriate, I think, but it depends on what is agreed towards your question:

> Should we have a third issue template for the third area: "suggestions for proposals to change or extend the CF conventions"?

I would support having separate issues for name(s) proposals and for amendments to the current table, because as well as allowing the titles to be more specific in each case we can adapt the template description a bit more to the specific case. And if we want to do fancy GitHub Actions workflows in future we could then have the option to trigger off the specific form of change (new name, or amendment).

So if there is agreement I think we should split this to two separate templates, else amend the name a little similar to the above suggestion to be explicit that either new names or changes can be proposed.